packages/web-runtime/src/container/bootstrap.ts

```diff
@@ -122,17 +122,17 @@ export const announceApplications = async ({
  * @param designSystem
  */
 export const announceTheme = async ({
-  themeLocation,
   store,
   vue,
-  designSystem
+  designSystem,
+  runtimeConfiguration
 }: {
-  themeLocation?: string
   store: Store<unknown>
   vue: VueConstructor
   designSystem: any
+  runtimeConfiguration?: RuntimeConfiguration
 }): Promise<void> => {
-  const { theme } = await loadTheme(themeLocation)
+  const { theme } = await loadTheme(runtimeConfiguration.theme)
```

Comment:
```suggestion
  const { theme } = await loadTheme(runtimeConfiguration?.theme)
```
since the `runtimeConfiguration` param is optional